obal/data/roles/build_srpm/tasks/main.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: 'Build SRPM'
+  command: "tito build --srpm --offline"
+  args:
+    chdir: "{{ inventory_dir }}/{{ package_base_dir }}/{{ inventory_hostname }}"
+  register: srpm_build
+
+- debug:
+    msg: "{{ srpm_build.stdout_lines }}"
+  when: srpm_build is successful
+
+- debug:
```

Comment:
This is never executed in the failure case, as ansible will abort after the `command` step failed.